src/Forms/GridFieldLinkButton.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * A button that contains a link to an URL.
+ *
+ * @package forms
+ * @subpackage fields-gridfield
+ */
+
+class GridFieldLinkButton implements GridField_HTMLProvider
+{
+
+    protected $targetFragment;
+    protected $link;
+
+    public function __construct($link, $targetFragment)
+    {
+        $this->link = $link;
+        $this->targetFragment = $targetFragment;
+    }
+
+    /**
+     * Place the link button in a <p> tag above the field
+     *
+     * @param GridField
+     *
+     * @return array
+     */
+    public function getHTMLFragments($gridField)
+    {
+        $button = "<a class='action gridfield-button-link ss-ui-button ui-button ui-widget ui-state-default ' .
+        ' ui-corner-all ui-button-text-icon-primary' href='$this->link'>".
+            "<span class='ui-button-icon-primary ui-icon ui-icon-search'></span> Find modules</a>";
```

Comment:
Please don't mix HTML into PHP. You can put this into a template and do something like this instead:

```php
$fragment = ArrayData::create([
    'Link' => $this->link,
    'Caption' => 'Find modules'
])->renderWith(__CLASS__);

return [$this->targetFragment => $fragment];
```

The "Find modules" sentence is hard coded, but the link is dynamic - can you please pass the text in as an argument as well?

Please also add localisation to the "Find modules" where you pass it in =)